aicsimageio/writers/ome_zarr_writer.py

```diff
@@ -249,26 +264,32 @@ def write_image(
             * image_data.shape[ydimindex]
             * image_data.itemsize
         )
+
         target_chunk_size = 16 * (1024 * 1024)  # 16 MB
         # this is making an assumption of chunking whole XY planes.
         # TODO allow callers to configure chunk dims?
         nplanes_per_chunk = int(math.ceil(target_chunk_size / plane_size))
         nplanes_per_chunk = (
             min(nplanes_per_chunk, image_data.shape[zdimindex]) if zdimindex > -1 else 1
         )
-        chunk_dim_map = {
-            "T": 1,
-            "C": 1,
-            "Z": nplanes_per_chunk,
-            "Y": image_data.shape[ydimindex],
-            "X": image_data.shape[xdimindex],
-        }
-        chunk_dims = [
+
+        if chunk_dims is None:
+            chunk_dim_map = {
+                "T": 1,
```

Comment:
Could you add a code comment here describing by T and C are 1?

Reply:
I believe this is because we only chunk by spatial dimensions, don't have a definitive explanation for this.